Deeploy/Targets/Generic/Parsers.py

```
@@ -2557,3 +2624,169 @@ def parseNodeCtxt(self,
+        # Common Inputs: 5 (X, scale, B, mean, var)
+        if len(node.inputs) < 5:
+            return False
+
+        # Save the attributes, default values are provided if not present
+        self.operatorRepresentation['epsilon'] = node.attrs.get('epsilon', 1e-5)
+        self.operatorRepresentation['momentum'] = node.attrs.get('momentum', 0.9)
+        self.operatorRepresentation['training_mode'] = node.attrs.get('training_mode', 0)
+
+        return True
+
+    def parseNodeCtxt(self, ctxt, node: gs.Node, channels_first: bool = True):
```

Ruff (0.12.2)

Unused method argument: `channels_first`

(ARG002)